libs/meta/include/meta/vm_types.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
!

This file does not belong in `libs/meta`.

Reply:
oh, sure, I have mistaken - it should be in ml/meta :)

Reply:
Also no. I discussed this with Khan - architecturally it is dangerous to couple high-level business logic like VM charge estimation to generic ML code, even if it's just with a few types. My recommendation is that ML should have an internal representation (even if it's just `using MLChargeAmount = uint64_t;`) for how much effort an operation took, and expose a way to that query. Then estimators in the `vm_modules` lib which know about `MLChargeAmount` and `vm::ChargeAmount` can sum up and convert the former into the latter for the virtual machine to use. This extra indirection avoids unnecessary coupling of ML code to VM charge estimation logic.

Reply:
agreed

Reply:
I have separated MLChargeAmount in /ml/ scope and ChargeAmount in /vm/ scope, both are uint64 under the hood.